pyproject.toml

```diff
@@ -68,16 +68,17 @@ max-complexity = 10
 
 [tool.mypy]
 files = [
+    "src/lightning",
     "src/pytorch_lightning",
     "src/lightning_fabric",
     "src/lightning_app",
```

Comment:
Remove this?

```suggestion
```

Reply:
There are still some files like `__main__.py` etc.